snapcraft.yaml

```diff
@@ -36,16 +27,7 @@ apps:
   simple-scan:
     plugs:
       - network
-      - network-control
       - home
-      - hardware-observe
```

Comment:
Are any of these actually needed for discovering USB scanners?

Reply:
Yes you are right. When I connect my printer via usb only, I needed to connect hardware-observe. Thanks! I'll see what is needed for a wired connection too.